Enchanted/UI/Shared/Sidebar/Components/ConversationHistoryListView.swift

```diff
@@ -62,6 +63,12 @@ struct ConversationHistoryList: View {
                 ForEach(conversationGroup.conversations, id:\.self) { dailyConversation in
                     HStack {
                         Button(action: {onTap(dailyConversation)}) {
+                            let isSelected = selectedConversation?.name == dailyConversation.name && selectedConversation?.createdAt == dailyConversation.createdAt
```

Comment:
Conversation doesn't have a unique ID so I'm using combination of `name` and `createdAt`. Not sure if having this conditional logic in view is acceptable here, but since it's quite simple (IMO) I think it's fine to put here.